.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -10,47 +10,46 @@ assignees: ''
 **Release**: [v0.x.x](https://github.com/GenericMappingTools/pygmt/milestones/?)
 **Scheduled Date**: YYYY/MM/DD
 **Pull request due date**: YYYY/MM/DD
+**DOI**: `10.5281/zenodo.XXXXXXX`
 
 **Priority PRs/issues to complete prior to release**
+
 - [ ] Wrap X ()
 - [ ] Wrap Y ()
 
 **Before release**:
+
 - [ ] Check [SPEC 0](https://scientific-python.org/specs/spec-0000/) to see if we need to bump the minimum supported versions of GMT, Python and core package dependencies (NumPy/Pandas/Xarray)
-- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
```

Comment:
Review the PyGMT team as early as possible so that we have more time for discussions.